packages/create-toolpad-app/src/writeCoreFiles.ts

```diff
@@ -0,0 +1,22 @@
+import path from 'path';
+import fs from 'fs/promises';
+
+export default async function writeCoreFiles(
```

Comment:
Is this function really specific to core? Seems quite general to me

```suggestion
export default async function writeFiles(
```